### DISCUSSION TITLE
Newbie: How to use local avatar/images in posts

### DISCUSSION THREAD

**Discussion by catusf** [cotes2020/jekyll-theme-chirpy#1521] [Q&A]
### Checklist

- [X] I have read and accepted the [contributing guidelines](https://github.com/cotes2020/jekyll-theme-chirpy/blob/master/docs/CONTRIBUTING.md).

### What is the topic?

Sharing tips and tricks

### Description

Hi all, a newbie here need your help
My site at https://catusf.github.io/ was working fine, last update was June 2023. 

Now I updated the theme to the latest version, all of my images disappeared from posts. For instance, instead of the image url of https://catusf.github.io//img/Pleco1.jpg, now it became  https://chirpy-img.netlify.app/img/Pleco1.jpg.

I played around with CDN settings, but no success.

What do I need to do to fix this?

Thanks a lot

**Comment by imagelessthought** [comment]
Did you resolve this?

I am looking at https://catusf.github.io/posts/T%E1%BB%AB-%C4%91i%E1%BB%83n-Trung-Vi%E1%BB%87t-cho-Pleco/.
